### PR TITLE
ROX-17200, ROX-17201: Send namespace and namespace labels for syslog alert notifications

### PR DESCRIPTION
**Pull request by theencee**
## Description

Send namespace name and namespace labels as part of the syslog alert notification if a new feature flag is enabled. 

## Checklist
- [ ] Investigated and inspected CI test results
- [x] Unit test and regression tests added
~[ ] Evaluated and added CHANGELOG entry if required~
~[ ] Determined and documented upgrade steps~
~[ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))~

If any of these don't apply, please comment below.

## Testing Performed

Manual and unit

In addition to reviewing your code, reviewers **must** also review your testing
instructions and make sure they are sufficient.

### CODE REVIEW

File: pkg/notifiers/metadata_getter.go [hunk 1]
```diff
@@ -11,4 +11,5 @@ import (
 //go:generate mockgen-wrapper MetadataGetter
 type MetadataGetter interface {
 	GetAnnotationValue(ctx context.Context, alert *storage.Alert, annotationKey, defaultValue string) string
+	GetNamespaceLabels(ctx context.Context, alert *storage.Alert) map[string]string
```
Comment (clickboo) [inline pkg/notifiers/metadata_getter.go:14]: GetLabelValue? Or is that not the intent - and the intent is return all labels?

Comment (theencee) [reply pkg/notifiers/metadata_getter.go:14]: Yes the request is to return _all_ of them

**Comment by theencee** [comment]
The ocp-4-12-qa-e2e-tests test failure looks unrelated.

**Review by clickboo — APPROVED** [review]
minor nit, lgtm otherwise

**Pull request merged**